parts/linux/cloud-init/artifacts/aks-check-network.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Check outbound connectivity from node to certain endpoints
+
+[Service]
+Type=oneshot
+ExecStart=/bin/bash /usr/local/bin/aks-check-network.sh
```

Comment:
we should probably specify network as a dependency

Reply:
`After=network-online.target`

Reply:
we'll also need to explicitly enable the unit as well - we should be able to do this during the VHD build itself (pre-install-dependenceis.sh)

Reply:
actually, it seems like we'd need the apiserver fqdn to actually invoke the script... so I guess we can't necessarily enable the unit during build time, would need to be at runtime during CSE instead

Reply:
Added dependency for network. I made the apiserver fqdn optional, so the script won't throw an error. But I think we still want to enable it at runtime so we can check connection to apiserver.